bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -209,6 +215,15 @@ public void operationComplete(int rc, Void result) {
             createComplete(BKException.Code.IncorrectParameterException, null);
             return;
         }
+
+        List<BookieSocketAddress> curEns = lh.getLedgerMetadata().getEnsemble(0L);
+        LOG.info("Ensemble: {} for ledger: {}", curEns, lh.getId());
```

Comment:
Change log level?

Reply:
We actually like it at info.  @jvrao thoughts?

Reply:
@eolivelli Would it be possible to leave it at info, or are there use cases you are worried about where the volume of ledger creation would be too high?

Reply:
Ok